spec/models/user_spec.rb

```diff
@@ -27,31 +24,19 @@
     end
```

Comment:
this test case `is valid with email attributes` is not needed as duplicate test case `is valid with valid attributes`